DigitalLearningSolutions.Web/Scripts/trackingSystem/centreCourseSetup.ts

```diff
@@ -0,0 +1,59 @@
+const copyCourseLinkClass = 'copy-course-button';
+const copyLinkIdPrefix = 'copy-course-';
+const launchCourseButtonIdPrefix = 'launch-course-';
+
+setupCourseLinkClipboardCopiers();
+
+function setupCourseLinkClipboardCopiers() {
+  const copyCourseLinks = Array.from(document.getElementsByClassName(copyCourseLinkClass));
+
+  copyCourseLinks.forEach(
+    (currentLink) => {
+      const linkId = currentLink.id;
+      const customisationId = linkId.slice(copyLinkIdPrefix.length);
+      currentLink.addEventListener('click', () => { copyLaunchCourseLinkToClipboard(customisationId); });

```

Comment:
Could we just do:
`currentLink.addEventListener('click', () => copyLaunchCourseLinkToClipboard(customisationId));`